docs: Add EBS CSI Driver permissions to 2.9 (#1454)
rancher/rancher-docs: versioned_docs/version-2.9/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md

}
```

### EBS CSI Driver addon Permissions

> Permissions required for Rancher to install the Amazon EBS CSI Driver add-on.

Reviewer: ```suggestion
The following are the required permissions for installing the Amazon EBS CSI Driver add-on.
```

}
```

### EBS CSI Driver addon Permissions

> The following are the required permissions for installing the Amazon EBS CSI Driver add-on.

Reviewer: ```suggestion
The following are the required permissions for installing the Amazon EBS CSI Driver add-on:
```
